services/utils/__init__.py

```diff
@@ -139,7 +140,7 @@ def __init__(self,
                  password: str = "postgres",
                  database_name: str = "postgres",
                  prefix="MF_METADATA_DB_"):
-        table = str.maketrans({"'": "\'", "`": "\`"})
+        table = str.maketrans({"'": r"\'", "`": r"\`"})
```

Comment:
has the original translation dictionary ever worked as intended? is there test coverage for this?

Reply:
not sure what you mean. We had a similar piece of code in migration service to deal with passing over the password to goose and it worked since those special characters needed to be escaped

Reply:
From a quick test the first item in the translation dictionary does not seem to escape anything.
translation dict prior to changes:
`{39: "'", 96: '\\`'}`

and after changes:
`{39: "\\'", 96: '\\`'}`

Might the change introduce some unintended behavior or is this the intended escaping behavior?